pkg/manager-server/file_system_lustre.go

```diff
@@ -171,10 +171,13 @@ func (f *FileSystemLustre) Mount(mountpoint string) error {
 		}
 	}
 
-	var devName string = f.devices[0]
+	var devName string 
 	if f.backFs == BackFsZfs {
 		devName = f.zfsVolName()
+	} else {
+		devName = f.devices[0]
```

Comment:
We're always sure that devices isn't nil or empty in this case?

Reply:
I don't think `NNF_SUPPLIED_DEVICES` is very robust in error handling - looks like it would also crash on the `Create()` call if empty so I'm not making it any worse off. Dean might know more.

Reply:
It's not robust.  It was a quick thing to get something working on my craystack-lop env.